src/languages/en.js

```diff
@@ -1072,4 +1072,30 @@ export default {
             message: 'Expensify does not have access to save attachments. To enable access, go to Settings and allow access',
         },
     },
+    desktopApplicationMenu: {
+        aboutExpensify: 'About New Expensify',
+        updateExpensify: 'Update New Expensify',
+        checkForUpdates: 'Check for updates',
```

Comment:
In the following image we can notice that only "Check for updates" and "View keyboard shortcuts" have small letter after space. Other options like "About New Expensify", "Hide New Expensify" have capital letter after space.
<img width="241" alt="Screenshot 2022-12-21 at 22 56 27" src="https://user-images.githubusercontent.com/25876548/208964224-dcabc1e8-8e5f-41d6-8471-d8e679838666.png">

Reply:
`New Expensify` is the name of the app so it's all uppercase. What I mean is that New is part of the name of the app. Pretty weird I know, but we will eventually call it Expensify without New.

Reply:
Ohh you mean in English the other options? It's funny because in Spanish, it does not do the same:

![image](https://user-images.githubusercontent.com/6474442/208965240-49ec6926-621f-4219-a8b5-d0e9aad68578.png)

Reply:
Either way, I changed it back because of this: https://github.com/Expensify/App/pull/13430#discussion_r1053651250

Reply:
I am going to localize all the options anyways, so it is not a huge deal since they will be all lowecase.